src/Facebook/Facebook.php

```diff
@@ -586,4 +589,70 @@ public function videoToUpload($pathToFile)
     {
         return new FacebookVideo($pathToFile);
     }
+
+    /**
+     * Upload video
+     *
+     * @param $target
+     * @param $pathToFile
+     * @param $accessToken
+     * @param int $maxTransferTries
+     *
+     * @return array
+     *
+     * @throws FacebookUploadException
+     */
+    public function uploadVideo($target, $pathToFile, $accessToken, $maxTransferTries = 5)
+    {
+        $uploader = $this->getResumableUploader($this->app, $accessToken, $maxTransferTries);
+        $endpoint = '/'.$target.'/videos';
```

Comment:
We don't want to guess the endpoint for them. They should be able to post to any endpoint.